agent/src/beerocks/slave/link_metrics/ieee802_3_link_metrics_collector.cpp

```diff
@@ -0,0 +1,279 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
It took me some time to understand that this is only for ethernet links. I guess I should know by heart that 802.3 is Ethernet, but consider changing the file name to `eth_link_metrics_collector.cpp` (nitpick)